spec/core_functions/math/abs.hrx

```diff
@@ -68,15 +74,17 @@ Recommendation: math.div(-7px, 4em)
 More info and automated migrator: https://sass-lang.com/d/slash-div
 
   ,
-1 | a {b: abs(-7px / 4em) * 1em}
-  |           ^^^^^^^^^^
+2 | a {b: math.abs(-7px / 4em) * 1em}
+  |                ^^^^^^^^^^
   '
-    input.scss 1:11  root stylesheet
+    input.scss 2:16  root stylesheet
 
 <===>
 ================================================================================
 <===> named/input.scss
-a {b: abs($number: -3)}
+@use "sass:math" as math;
+$number: -3;
+a {b: math.abs($number)}
```

Comment:
This isn't actually testing a named argument; it's testing passing a variable called `$number` to `abs`.

To test this with the function in the math module, this should be:
```scss
@use "sass:math";
a { b: math.abs($number: 3)}
```

However, since this is a case where even the global `abs` should still parse as a Sass function, the existing test of the global function (`abs($number: -3)`) should also be kept.

Reply:
Nice catch, I was missing the try/catch in the abs function.